src/main/scala/redis/Sentinel.scala

```diff
@@ -138,6 +178,23 @@ abstract class SentinelMonitored(system: ActorSystem) {
 
     Await.result(ff, 15 seconds)
   }
+
+  def withSlavesAddr[T](initFunction: Seq[(String, Int)] => T): T = {
+     import scala.concurrent.duration._
+
+    val fslaves = Future.sequence(sentinelClients.values.map(_.slaves(master)))
+      .map { lm =>
+        val ipAndPort = lm.flatten.flatMap { m =>
+          (m.get("ip"), m.get("port")) match {
+            case (Some(ip), Some(port)) => Some((ip, port.toInt))
+            case _ => None
+          }
+        }.toSeq.distinct
+        initFunction(ipAndPort)
+      }
+    Await.result(fslaves, 15 seconds)
```

Comment:
Hum I don't like the Await (blocking)
I would prefer this function : 
`def slaves():Future[(String,Int)]`

I will try to refactor `withMasterAddr` so there is no Await #108